pkgs/applications/networking/irc/irssi/otr/default.nix

```diff
@@ -1,16 +1,14 @@
 { stdenv, fetchurl, libotr, automake, autoconf, libtool, glib, pkgconfig, irssi }:
 
-let
-  rev = "640e98c74b";
-in
 with stdenv.lib;
 stdenv.mkDerivation rec {
-  name = "irssi-otr-20131007-${rev}";
-  
+  name = "irssi-otr-${version}";
+  version = "1.0.0";
+
   src = fetchurl {
```

Comment:
Please use `fetchFromGitHub` instead. Github sometimes rebuilds the tarballs which causes the hash to change.